craco.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  style: {
+    postcss: {
+      plugins: [require("tailwindcss"), require("autoprefixer")],
+    },
+  },
+  babel: {
+    plugins: ["@emotion"],
```

Comment:
what is this plugin

Reply:
This is the [emotion babel plugin](https://www.npmjs.com/package/@emotion/babel-plugin) and is recommended when using emotion and styled components. I've added it because I needed to use Styled Components as CSS selectors and that is not possible without the plugin. 
For example with the plugin you can do things like 
```javascript 
const Link = styled.a`//... some styles`
const Navbar = styled.nav`
// ... some other styles 
& ${Link} {
   // All Links components inside the Navbar will have text red
   color: red;
}
`
```